recovery_behavior/include/recovery_behavior/plugins/action/recover_pose_action.hpp

```diff
@@ -0,0 +1,98 @@
+// Licensed under MIT. See LICENSE file. Copyright Carologistics.
+
+// Copyright (c) 2018 Intel Corporation
```

Comment:
remove long license header, keep line 1

also, why is there Intel Corporation?!!?

Reply:
fixed, pls refer to latest commit - dce5e3e